src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -137,6 +137,21 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
         t.transactionResponse.map(Extraction.decompose).getOrElse(JNull)
     }
 
+  implicit val eth_d  =
+    new JsonDecoder[GetTransactionByBlockNumberAndIndexRequest] with JsonEncoder[GetTransactionByBlockNumberAndIndexResponse] {
+      override def decodeJson(params: Option[JArray]): Either[JsonRpcError, GetTransactionByBlockNumberAndIndexRequest] = params match {
+        case Some(JArray(JString(blockStr) :: transactionIndex :: Nil)) =>
```

Comment:
Should we use `blockStr: JString` instead of `JString(blockStr)` to allow `blockStr` to contain a `JInt` (as done for example in `eth_getBalance`)?